basicUserProfile.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$id": "https://getdock.github.io/schemas/basicUserProfile.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "Basic user profile.",
+  "title": "Basic User Profile",
+  "allOf": [
+    {
+      "$ref": "https://getdock.github.io/schemas/dock.json"
+    },
+  "type": "object",
+  "properties": {
```

Comment:
These properties should be within `$data` object. This applies for all schemas updated here.

```json
"type": "object",
"properties": {
    "$data": {
        "type": "object",
        "properties": {
             "firstName": {....}
        }
    }
}
```